json-endpoints/jetpack/class.jetpack-json-api-sync-endpoint.php

```diff
@@ -324,3 +324,36 @@ protected function result() {
 		);
 	}
 }
+
+class Jetpack_JSON_API_Sync_Object_Id_Range extends Jetpack_JSON_API_Sync_Endpoint {
+	protected function result() {
+		$args = $this->query_args();
+
+		$module_name = $args['sync_module'];
+		$batch_size  = $args['batch_size'];
+
+		if ( ! $this->is_valid_sync_module( $module_name ) ) {
+			return new WP_Error( 'invalid_module', 'This sync module cannot be used to calculate a range.', 400 );
+		}
+
+		$module = Modules::get_module( $module_name );
+
+		return array(
+			'ranges' => $module->get_min_max_object_ids_for_batches( $batch_size ),
+		);
+	}
+
+	protected function is_valid_sync_module( $module_name ) {
```

Comment:
This method's name is a bit confusing. There's a bunch of sync modules that we don't support here. How about we rename the method to `is_valid_sync_object_module` or even `is_supported_sync_module`?